include/SDL3/SDL_stdinc.h

```diff
@@ -80,7 +80,7 @@ char *alloca();
 /**
  *  The number of elements in an array.
  */
-#define SDL_arraysize(array)    (sizeof(array)/sizeof(array[0]))
+#define SDL_arraysize(array)    ((int)(sizeof(array)/sizeof(array[0])))
```

Comment:
I think this cast changes the return type's size from 64 bits to 32 bits on 64 bit systems (along with making it signed.)

Reply:
it's meant to be marked signed because it's subject to arithmetic.
it's a compile time function, it doesn't seem to be possible to do so big array a compile time.